source/array-to-union.d.ts

```diff
@@ -0,0 +1,29 @@
+/**
+Convert a fixed-size array into a union type of its elements.
```

Comment:
"fixed size array", isn't this a tuple? And then the name should probably be `TupleToUnion`.

Reply:
Yeah, I went back-and-forth on using tuple vs array